src/modules/displayCard.js

```diff
@@ -0,0 +1,36 @@
+import { getItemLikes } from './likeApi.js';
+import initializeEventHandlers from './eventHandlers.js';
+
+const showsList = document.getElementById('shows-list');
+
+const init = async (Data) => {
+  const likesPromises = Data.map(async (item) => {
+    const likes = await getItemLikes(item.id);
+    const numOfLike = likes.find((like) => like.item_id === item.id);
+    return { item, numOfLike };
+  });
+
+  const itemsWithLikes = await Promise.all(likesPromises);
+
+  itemsWithLikes.forEach(({ item, numOfLike }) => {
+    const showCard = document.createElement('div');
+    showCard.className = 'show-card';
+    showCard.dataset.id = item.id;
+
+    showCard.innerHTML = `
+      <img class="show-image" src="${item.image.medium}">
+      <div class="show-title">${item.name}</div>
+      <div class="box-like">
+        <button class="btn-like"><span>👍</span></button>
+        <span class="counter-like">${numOfLike ? numOfLike.likes : 0}</span>
+      </div>
+      <button class="btn btn-comment">Add Comment</button>
+    `;
+
+    showsList.appendChild(showCard);
```

Comment:
- [ ] Well done on successfully fetching data from the API and rendering it on the home page! It's evident that you've met the initial requirements effectively. However, there's one additional feature that I'd recommend implementing to enhance the user experience further.

   - Considering the project requirements, I suggest adding a `counter` functionality that calculates the total number of items available on the home page and displays it on the user interface. This addition will provide valuable information to users, making the page more informative. See below for more details on project requirements:
<img width="534" alt="Screenshot 2023-08-04 at 20 05 38" src="https://github.com/jlcarrascof/JS-Capstone-Project/assets/107192076/bce2391d-c267-45f2-b9fe-299d70a317ab">